libraries/ESP8266WiFi/src/WiFiClientSecureBearSSL.h

```diff
@@ -104,9 +105,14 @@ class WiFiClientSecure : public WiFiClient {
     static bool probeMaxFragmentLength(const char *hostname, uint16_t port, uint16_t len);
     static bool probeMaxFragmentLength(const String host, uint16_t port, uint16_t len);
 
-    // AXTLS compatbile wrappers
-    bool verify(const char* fingerprint, const char* domain_name) { (void) fingerprint; (void) domain_name; return false; } // Can't handle this case, need app code changes
+    // AXTLS compatible wrappers
     bool verifyCertChain(const char* domain_name) { (void)domain_name; return connected(); } // If we're connected, the cert passed validation during handshake
```

Comment:
Does it make sense to verify a certificate chain before connection? How does verification work with axTLS?
My question is aimed at whether a connection should be initiated here in case it is not established.

Reply:
No, it doesn't make sense before a connection.  You get a cert chain as part of a connection handshake.  In axTLS this is just stuffed away and not examined unless you call .verify() at some later point.  In BearSSL it's verified as part of the connection setup and if it fails there is no connection.  If you connected, verified, then disconnected, there is no guarantee you'd connect to the same server on the next real .connect().

The only reason `verify()` was there in the initial BearSSL was for compatibility with axTLS code (at least compilation...it would fail when run if there was no prior `setFingerprint()` type call.  If you want to rename it to something else, I'd say drop it as it's always going to be a no-op in BearSSL.  Suggest dropping this or using original method code with a `return connected()` instead of `return false`.

Reply:
@earlephilhower I'm a bit confused. My question was for `verifyCertChain()`, which already returns connected().